src/app/frontend/logs/logs.html

```diff
@@ -55,34 +55,47 @@
                  ng-class="ctrl.getColorIconClass()">
           format_color_text
         </md-icon>
+        <md-tooltip>
+          [[Invert colors|Toggles between light & dark background in log viewer.]]
+        </md-tooltip>
       </md-button>
       <md-button class="kd-logs-toolbar-button"
                  ng-click="ctrl.onFontSizeChange()">
         <md-icon md-font-library="material-icons"
                  ng-class="ctrl.getSizeIconClass()">
           text_fields
         </md-icon>
+        <md-tooltip>
+          [[Toggle font size|Toggles between a larger and smaller font option in the log viewer.]]
+        </md-tooltip>
       </md-button>
       <md-button class="kd-logs-toolbar-button"
                  ng-click="ctrl.onShowTimestamp()">
         <md-icon md-font-library="material-icons"
                  class="kd-logs-icon">
           {{ctrl.getTimestampIcon()}}
         </md-icon>
+        <md-tooltip>
+          [[Toggle timestamps|Toggles between hiding and showing timestamps for log items.]]
+        </md-tooltip>
       </md-button>
       <md-button class="kd-logs-toolbar-button"
                  ng-click="ctrl.onPreviousChange()">
         <md-icon md-font-library="material-icons"
                  class="kd-logs-icon">
           {{ctrl.getPreviousIcon()}}
         </md-icon>
+        <md-tooltip>{{ctrl.getPreviousTooltip()}}</md-tooltip>
```

Comment:
Use `ng-if` here. If possible we are trying to keep all translations directly in html.

Reply:
I _think_ I've made the change you've requested, but I'm not an Angular dev. Please let me know if it's now more of the standard.